types/types.go

```diff
@@ -907,17 +907,17 @@ const (
 )
 
 // MergeAssetConfig merges together two asset param objects.
-func MergeAssetConfig(old, new sdk_types.AssetParams) (out sdk_types.AssetParams) {
+func MergeAssetConfig(old, new AssetParams) (out AssetParams) {
 	// if asset is new, set.
 	// if new config is empty, set empty.
 	// else, update.
-	if old == (sdk_types.AssetParams{}) {
-		out = new
-	} else if new == (sdk_types.AssetParams{}) {
+	if old.IsZero() {
 		out = new
+	} else if new.IsZero() {
+		out = old
 	} else {
 		out = old
-		if !old.Manager.IsZero() {
+		if !old.IsZero() {
```

Comment:
I'm confused about this change. Could you explain?

Reply:
Thanks, I shouldn't have removed `Manager`. Technically these checks are optional anyway because the transactions have already been validated.